### ISSUE TITLE
Angular 4 template : generateInterface doesn't work

### ISSUE THREAD

**Issue by mvidailhet** [geeklearningio/gl-swagger-generator#66]
If I set the generateInterface option to true in the templateOptions, the Interface file is not generated.
In the client file, I have "implements IMyService", but the interface IMyService is not generated.

**Comment by sandorfr** [comment]
true, support for interface has not been implemented in that template. What do you need it for?

**Comment by mvidailhet** [comment]
I don't, but it creates a bug when I set the param to true, as it implements an interface that doesn't exist

**Comment by sandorfr** [comment]
yeah but don't set it to true then :)
